projects/extension/sql/idempotent/005-chunking.sql

```diff
@@ -2,7 +2,7 @@
 -------------------------------------------------------------------------------
 -- chunking_character_text_splitter
 create or replace function ai.chunking_character_text_splitter
-( chunk_column pg_catalog.name
+( chunk_column pg_catalog.name default ''
```

Comment:
i think this column will go away completely when we have a `loader_row` configuration, but in case it doesn't the default should be `null` rather than `''`